doc/migrations/3.1.1_3.1.1++.md

```diff
@@ -0,0 +1,8 @@
+## **`odbc` renamed to `rdbms` in module names and options**
+
+* **For MongooseIM users:** simply replace all instances of `odbc` in your config files with `rdbms`.
+  E.g. `{odbc_server, {mysql, "server", 1234, "database", "username", "password"}}.` would become `{rdbms_server, {mysql, "server", 1234, "database", "username", "password"}}.`, and `{auth_method, odbc}.` would now be `{auth_method, rdbms}.`.<br/>
+  It's also important to note that all metrics that previously contained `odbc` in their names have also been renamed to contain `rdbms` instead.
+
+* **For developers calling MongooseIM modules:** most modules, functions and atoms had `odbc` in their names replaced with `rdbms`.
+  The only exceptions to this rule were names actually pertaining to ODBC driver, e.g. `mongoose_rdbms_odbc` or `ejabberd_check:check_database_module(odbc)`.
```

Comment:
The only exceptions to this rule were names actually pertaining to the ODBC driver, e.g. `mongoose_rdbms_odbc` or `ejabberd_check:check_database_module(odbc)`.